Fundamentals/Backoffice/Property-Editors/Built-in-Property-Editors/Block-Grid-Editor/index.md

```diff
@@ -0,0 +1,670 @@
+
+
+### Catalogue appearance
+
+These properties refer to how the Block is presented in the Block catalogue when editors choose which Blocks to use for their content.
+
+- **Background color** - Define a background color to be displayed beneath the icon or thumbnail. Example: `#424242`.
+- **Icon color** - Change the color of the Element Type icon. Example: `#242424`.
+- **Thumbnail** - Pick an image or SVG file to replace the icon of this Block in the catalogue.
+
+The thumbnails for the catalogue are presented in the format of 16:10, and we recommend a resolution of 400px width and 250px height.
+
+### Allowance
+
+
+- **Allow in root color** - Determine wether this Block can be created in the root of your layout.
+- **Allow in area** - Determine wether this Block can be created inside Areas of other Blocks.
+
+
+### Areas
+
+Block Areas enables for nesting of Blocks.
+The each area has a size, defined by column and rows spans.
+The grid for the Areas are based on the same amount of columns as your root grid, unless you choose to change it.
+To scale a Area drag the little scale-button in the bottom right corner with your mouse.
+
+- **Grid Columns for Areas** - Overwrite the amount of columns used for the grid of Areas.
+- **Allow in area** - Determine wether this Block can be created inside Areas of other Blocks.
+
+### Area configuration
+
+> Missing screenshot of Area configuration
+
+**MISSING DESCRIPTION OF AREA CONFIG**
+Mention that the alias can be used to target this Area via the Layout Stylesheet.
+
+**Create Button Label** is to overwrite the Create Button Label of that Area.
+
+**Number of blocks** is for the total amount of Blocks in the Area.
+
+**TODO...**
+Mention that you can pick groups for allowance, in that case the required number is for Blocks of the group.
+Mention that you can pick individual Blocks, this Block can be picked and will be valid though its not allow for Areas.
+Individual Block validation(required number) is just for that block-type.
+
+
+### Advanced
+
+These properties are relevant when you working with custom views or complex projects.
+
+- **Custom view** - Overwrite the AngularJS view for the block presentation in the Content editor. Use this to make a more visual presentation of the block or make your own editing experience by adding your own AngularJS controller to the view.
+
+- **Custom stylesheet** - Pick your own stylesheet to be used for this block in the Content editor. By adding a stylesheet the styling of this block will become scoped. This means that the default backoffice styles are no longer present for the view of this block.
+
+- **Overlay editor size** - Set the size for the Content editor overlay for editing this block.
+
+- **Hide content editor** - Use this feature when you want to hide the ability to edit the content in a Block Editor. If you made a custom view that enables you to edit the content part of a Block, you might want to use this feature.
```

Comment:
I am not sure the copy for this @nielslyngsoe & @sofietoft reads right or is super clear, but may just be me...

Reply:
Okay, I changed it to this:

Hide the UI for editing the content in a Block Editor. This is only relevant if you made a custom view that provides the UI for editing of content.